### DISCUSSION TITLE
Improve message of the day

### DISCUSSION THREAD

**Discussion by beliven-fabrizio-gortani** [gravitational/teleport#22277] [Ideas]
Hi everyone! I know that it's not an essential feature for the functionalities offered by the fantastic Goteleport, but if it doesn't require a huge implementation effort, it would be nice if the system could read the message of the day from a file, so that it can generate nicer and more personalized messages (such as multi-line or ASCII art).

**Comment by webvictim** [comment]
I've converted this to a feature request for you: https://github.com/gravitational/teleport/issues/22334
